viewer/packages/tools/src/Measurement/types.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+export type MeasurementLineOptions = {
+  lineWidth?: number;
+  color?: number;
+};
+
+export type MeasurementUnitUpdateDelegate = () => MeasurementUnits;
+
+export type MeasurementOptions = {
+  unitsUpdateCallback?: MeasurementUnitUpdateDelegate;
```

Comment:
I think a better way would be to have a delegate that just transforms the label. By this I mean that we accept a delegate f.ex.
```ts
transformMeasurementLabel(distanceMeter: number): string;
```

Then the user can transform from meters -> whatever they want themselves and so we would not have to worry about supporting all kinds of measurement units. Also we would then not have to deal with formatting (as this is now all user side). What do you think?

PS: I am not very happy with the names of this delegate (both function name and parameter), so see if you can find something better